### ISSUE TITLE
Create DELETE APIs

### ISSUE THREAD

**Issue by aviaryan** [fossasia/open-event-orga-server#510]
See #336 for the schema

Currently these POST methods will be publicy accessible (login needed). Only after #429 is done (which in on hold currently), the access control can be added to it.

**Comment by mariobehling** [comment]
Please assign all issues. Cheers.

**Comment by aviaryan** [comment]
closing this as new issue #597 has been created for DELETE APIs
